src/MainUtils.cpp

```diff
@@ -223,6 +223,11 @@ void genConstPackObj(const mlir::OwningModuleRef &module,
       .exec();
 
 #else
+  /* FIXME: final object file should be set in constPackObjPath
```

Comment:
Hi Gong, I'm not seeing what needs fixing. The patch presented here seems to resolve the issue fully?

Also, the constant pack for platforms that are **not** Linux or Mac shouldn't be deleted. This is because baking the parameters into the shared library is a very platform-specific process, and I only coded up the mac & Linux version; for platforms like Windows, we resort to having separate files for shared library and parameter constant packs.

So to keep the naming consistent between the constant pack & the shared library, they will share the same name (conveyed via `outputBaseName`) except for the suffix (e.g., `mnist.so` for shared library and `mnist.bin` for parameters) on platforms like Windows.

Reply:
@tjingrant Great. That's exactly why you need to take a lot and make sure that my fix didn't just work by luck. I was obviously under the wrong impression that the constant pack on Windows should be deleted once it's embedded into the shared library like on Linux and MacOS. I will update the comment to reflect this. Thanks.